### ISSUE TITLE
Error in Godot 3D documentation

### ISSUE THREAD

**Issue by Overhaul66** [godotengine/godot#87955]
### Tested versions

*Godot 4.2 documentation*
Transform 3D was explained incorrectly

![IMG-20240204-WA0016](https://github.com/godotengine/godot/assets/89018580/e1134b56-72ec-4772-8988-22877bb98625)

![IMG-20240204-WA0015](https://github.com/godotengine/godot/assets/89018580/dd0bd371-9e52-4f84-885d-a406c07581cf)

### System information

Internet

### Issue description

There was a mistake in explaining the matrix that transform3D represents

![IMG-20240204-WA0016](https://github.com/godotengine/godot/assets/89018580/53adbd9d-4848-4752-bee3-18e7691700c7)

A transform3D is supposed to be a 3x4 matrix

![IMG-20240204-WA0015](https://github.com/godotengine/godot/assets/89018580/a3e84acf-705b-4299-87fc-4c39c31cdad1)

### Steps to reproduce

None

### Minimal reproduction project (MRP)

None

**Comment by brcontainer** [comment]
I may be wrong, but I believe that the documentation is community, but separately, you should send corrections to the documentation as advised in:

https://docs.godotengine.org/en/latest/contributing/documentation/contributing_to_the_documentation.html

Related: https://github.com/godotengine/godot-docs

**Comment by Mickeon** [comment]
> I may be wrong, but I believe that the documentation is community, but separately, you should send corrections to the documentation as advised in:

This is true, but the contents of the class reference are actually [generated from this repository](https://github.com/godotengine/godot/blob/master/doc/classes/Transform3D.xml). So it's appropriate to report it here.

If factually correct, I can keep this in mind for https://github.com/godotengine/godot/pull/87334

**Comment by Mickeon** [comment]
I found the "offending" paragraph https://github.com/godotengine/godot-docs/blob/62d5d6cddbb2768976abea1cd3420b36ca492003/tutorials/3d/introduction_to_3d.rst#L35 as part of the introduction to 3D. Yes, this issue should be moved to the other repository.
